ClubManagement/ClubManagement/Activities/CreateFeeActivity.cs

```diff
@@ -10,6 +12,43 @@ namespace ClubManagement.Activities
     [Activity(Label = "CreateFeeActivity")]
     public class CreateFeeActivity : CreateOrEditFeeActivity
     {
+        [InjectOnClick(Resource.Id.btnDone)]
+        private void Done(object sender, EventArgs e)
+        {
+            if (string.IsNullOrEmpty(edtDescription.Text) ||
+                string.IsNullOrEmpty(edtChooseFeeGroup.Text) ||
+                string.IsNullOrEmpty(edtDeadline.Text) ||
+                !long.TryParse(edtAmount.Text, out long amount) || amount == 0)
+            {
+                Toast.MakeText(this, GetString(Resource.String.please_fill_all_fields), ToastLength.Short).Show();
```

Comment:
Use `this.ShowMessage(Resource.String.please_fill_all_fields)` please

Reply:
Done